dev-tools/validation/test_check_authz_route_matrix.py

```
@@ -3,125 +3,1484 @@
+
+
+def route_for(document, service, route_name):
+    matches = [
+        route
+        for route in document["routes"]
+        if route.get("service") == service and route.get("route") == route_name
+    ]
+    if len(matches) != 1:
+        raise AssertionError(
+            f"expected exactly one {service} {route_name} route, found {len(matches)}"
+        )
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)